tests/integ/test_dataframe.py

```diff
@@ -804,6 +806,24 @@ def test_explode(session):
         df.select(df.strs, explode(df.maps).as_("primo", "secundo")), expected_result
     )
 
+    # with input as array construct
+    Utils.check_answer(
+        session.range(1).select(explode(array_construct(lit(1), lit(2), lit(3)))),
+        [Row(VALUE="1"), Row(VALUE="2"), Row(VALUE="3")],
+    )
+
+    try:
+        context._use_structured_type_semantics = True
```

Comment:
I'd recommend setting this with a patch context. I've seen other tests fail due to various context flags being set when running tests in parallel.
```suggestion
        with patch.object(context, "_use_structured_type_semantics", True):
```